src/Components/profileOutline.js

```diff
@@ -0,0 +1,181 @@
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Phone:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].phone}
+    </div> 
+        <br>
+        </br>
+    <div style={styles.header}>
+        City:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].city} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        State:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].state} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Ethnicity:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].ethnicity} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        Age:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].age}
+    </div> 
+        <br>
+        </br>
+    <div style={styles.header}>
+        Gender Identity:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].genderIdentity} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        Sexuality:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].sexuality} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Veteran:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].veteran} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Disability:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].disability}
+    </div> 
+        <br>
+        </br>
+    <div style={styles.header}>
+        Housing Situation:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].housingSituation} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        Employment Status:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].employmentStatus} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Prior Convictions:
+    </div>
+    <div style={styles.text}>
+        {demographic_info[0].priorConvictions} 
+    </div>
+        <br>
+        </br>
+</div>
+
+)
+
+
+}
+
+ProfileOutline.propTypes = {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    pronouns: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    phone: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    state: PropTypes.string.isRequired,
+    ethnicity: PropTypes.string.isRequired,
+    age: PropTypes.number.isRequired,
+    genderIdentity: PropTypes.string.isRequired,
+    sexuality: PropTypes.string.isRequired,
+    veteran: PropTypes.string.isRequired,
+    disability: PropTypes.string.isRequired,
+    housingSituation: PropTypes.string.isRequired,
+    employmentStatus: PropTypes.string.isRequired,
+    priorConvictions: PropTypes.string.isRequired,
+  }
```

Comment:
This looks a lottt cleaner good job buds!! I think one more thing we can work to change before pushing is the html setup of the component! Until we have the designs we don't really need to worry about styling. That being said, typically most of the styling should be done in css, so things like the <br> tag can be replaced with spacing, etc. I think for now you have a couple options. You could 
a) create another component that has the html setup for all of these different demographic info which we talked about earlier, so the profile component element would have a demographics component

b) you could probably consolidate a lot of this text into one div with some text elements which would look something like 
<div>
<p>Name: {demographics.name}</p>
....
<p>Veteran Status: {demographics.status}</p>
</div>
stellar performance as always 😁